app/controllers/ctc/update/dependents_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Ctc::Update::DependentsController < Ctc::Portal::DependentsController
+  def confirm_remove
+    render 'ctc/portal/dependents/confirm_remove'
+  end
+
+  def destroy
+    current_model.destroy!
+    flash[:alert] = "Family member / dependent removed from your tax return."
+    redirect_to questions_confirm_information_path
+  end
+
+  def update
```

Comment:
i think we'd still have to do this because the base revisions controller expects to create a note and a couple other things, which we're bypassing here. I could have this controller inherit from baserevisions and then pull in the stuff that makes the dependents controller unique here instead, but this ended up being the more straightforward change.